<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>786</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>351</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="usernameText">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #eaf4ff;                  /* Subtle blue-tinted background */
    color: #333;                                /* Dark text color */
    border: 1px solid #0078D4;                  /* Blue border */
    border-radius: 8px;                         /* Rounded corners for consistency */
    padding: 10px 20px;                         /* Inner padding for a comfortable size */
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 18px;                            /* Modern text size */
    min-height: 40px;                           /* Min height for consistency with buttons */
    box-shadow: 0px 2px 4px rgba(0, 120, 212, 0.2); /* Subtle shadow for depth */
    transition: border-color 0.3s ease, box-shadow 0.3s ease; /* Smooth transitions */
}

QLineEdit:focus {
    border: 1px solid #005A9E;                  /* Slightly darker blue on focus */
    background-color: #ffffff;                  /* White background for better focus visibility */
    color: #1c1c1e;                             /* Darker text color */
    box-shadow: 0 0 6px rgba(0, 90, 158, 0.4);  /* Subtle blue glow around input */
}

QLineEdit::placeholder {
    color: #a6a6a6;                             /* Light gray placeholder text */
    font-style: italic;                         /* Italic placeholder */
}</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="passwordText">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #eaf4ff;                  /* Subtle blue-tinted background */
    color: #333;                                /* Dark text color */
    border: 1px solid #0078D4;                  /* Blue border */
    border-radius: 8px;                         /* Rounded corners for consistency */
    padding: 10px 20px;                         /* Inner padding for a comfortable size */
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 18px;                            /* Modern text size */
    min-height: 40px;                           /* Min height for consistency with buttons */
    box-shadow: 0px 2px 4px rgba(0, 120, 212, 0.2); /* Subtle shadow for depth */
    transition: border-color 0.3s ease, box-shadow 0.3s ease; /* Smooth transitions */
}

QLineEdit:focus {
    border: 1px solid #005A9E;                  /* Slightly darker blue on focus */
    background-color: #ffffff;                  /* White background for better focus visibility */
    color: #1c1c1e;                             /* Darker text color */
    box-shadow: 0 0 6px rgba(0, 90, 158, 0.4);  /* Subtle blue glow around input */
}

QLineEdit::placeholder {
    color: #a6a6a6;                             /* Light gray placeholder text */
    font-style: italic;                         /* Italic placeholder */
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QPushButton" name="RegisterButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D4; /* Modern Fluent Blue */
    color: white;
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 18px; /* Slightly larger text size */
    font-weight: 600;
    padding: 10px 22px; /* Increased padding for a larger button size */
    border: none;
    border-radius: 8px; /* Rounded corners for a modern look */
    min-height: 40px;
    min-width: 120px;
    box-shadow: 0px 4px 6px rgba(0, 120, 212, 0.2); /* Subtle shadow for depth */
    transition: background-color 0.3s ease, box-shadow 0.3s ease; /* Smooth transitions */
}

QPushButton:hover {
    background-color: #005A9E; /* Slightly darker blue on hover */
    box-shadow: 0px 6px 8px rgba(0, 90, 158, 0.3); /* Enhanced shadow on hover */
}

QPushButton:pressed {
    background-color: #003A6D; /* Even darker blue when pressed */
    box-shadow: 0px 2px 4px rgba(0, 58, 109, 0.2); /* Reduced shadow when pressed */
}

QPushButton:disabled {
    background-color: #A6A6A6; /* Gray for disabled state */
    color: #FFFFFF; /* Keep white text for contrast */
    box-shadow: none;
    cursor: not-allowed;
}</string>
         </property>
         <property name="text">
          <string>Sign up</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="LoginButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0078D4; /* Modern Fluent Blue */
    color: white;
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 18px; /* Slightly larger text size */
    font-weight: 600;
    padding: 10px 22px; /* Increased padding for a larger button size */
    border: none;
    border-radius: 8px; /* Rounded corners for a modern look */
    min-height: 40px;
    min-width: 120px;
    box-shadow: 0px 4px 6px rgba(0, 120, 212, 0.2); /* Subtle shadow for depth */
    transition: background-color 0.3s ease, box-shadow 0.3s ease; /* Smooth transitions */
}

QPushButton:hover {
    background-color: #005A9E; /* Slightly darker blue on hover */
    box-shadow: 0px 6px 8px rgba(0, 90, 158, 0.3); /* Enhanced shadow on hover */
}

QPushButton:pressed {
    background-color: #003A6D; /* Even darker blue when pressed */
    box-shadow: 0px 2px 4px rgba(0, 58, 109, 0.2); /* Reduced shadow when pressed */
}

QPushButton:disabled {
    background-color: #A6A6A6; /* Gray for disabled state */
    color: #FFFFFF; /* Keep white text for contrast */
    box-shadow: none;
    cursor: not-allowed;
}</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background: transparent;
    color: #1c1c1e;
    font-family: &quot;San Francisco&quot;, &quot;Helvetica Neue&quot;, &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 24px;
    font-weight: 500;
    letter-spacing: 0.5px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameText</tabstop>
  <tabstop>passwordText</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>RegisterButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
